docs/source/release/v6.7.0/Reflectometry/New_features/35494.rst

```diff
@@ -0,0 +1 @@
+- The ReflectometryISISCalibration algorithm now calculates and performs individual detector pixel corrections from an input file containing detector IDs and theta offsets.
```

Comment:
```suggestion
- The :ref:`algm-ReflectometryISISCalibration` algorithm now calculates and performs individual detector pixel corrections from an input file containing detector IDs and theta offsets.
```

Reply:
Will that work OK if the documentation doesn't exist yet? I'd deliberately left it out for now assuming I would need to add it in a later PR that adds the docs, but let me know if I'm misunderstanding how this works.